app.js

```diff
@@ -10,16 +11,22 @@ export default class App extends Component {
       data: [],
       question: '',
       answer: '',
-      view: '#cards'
+      view: window.location.hash,
+      isEditing: false,
+      editId: null
```

Comment:
If this is an `index`, name it accordingly. Do not mistake an item's position with its identity.